packages/tendermint-rpc/src/types.ts

```diff
@@ -22,3 +22,18 @@ export interface ValidatorEd25519Signature {
  * Currently only Ed25519 supported.
  */
 export type ValidatorSignature = ValidatorEd25519Signature;
+
+export enum BlockIdFlag {
+  BlockIdFlagUnknown = 0,
+  BlockIdFlagAbsent = 1,
+  BlockIdFlagCommit = 2,
+  BlockIdFlagNil = 3,
```

Comment:
Do we really want to releat the enum name in the cases? As far as I can tell, we must write them anyways (`BlockIdFlag.BlockIdFlagCommit`)